example/example.php

```diff
@@ -18,7 +18,8 @@
 use AbcAeffchen\SepaUtilities\SepaUtilities;
 use AbcAeffchen\Sephpa\SephpaDirectDebit;
 
-require_once '../src/Sephpa.php';
+chdir(dirname(__DIR__));
+require __DIR__ . '/../vendor/autoload.php';
```

Comment:
The vendor folder is not part of the repository. There should be a note, that is requires composer. Additionally a comment with instructions to use this without composer would be nice. This is basically using  src/autoloader.php and SepaUtilities.php that needs to be downloaded separately.